src/testdir/test_quickfix.vim

```diff
@@ -6370,7 +6370,7 @@ func Test_loclist_replace_autocmd()
   call setloclist(0, [], 'f')
   let s:bufnr = bufnr()
   cal setloclist(0, [{'0': 0, '': ''}])
-  au BufEnter * cal setloclist(1, [{'t': ''}, {'bufnr': s:bufnr}], 'r')
+  au BufEnter * cal setloclist(1, [{'t': ''}, {'bufnr': s:bufnr}, {'t': ''}], 'r')
```

Comment:
This change is very non-obvious:

- This test replaced the loclist every time BufEnter occurs.
- Now that we preserve the loclist location when the loclist is being replaced, we *stay* on the selected item when selected it. Previously, we'd select the second item, which would switch buffers and trigger replacement of the loclist, which would then reset our selected loclist index to 1.
- So, we stay on entry 2, and then we call `lnext`. This fails because there isn't a 3rd entry to navigate to.

The easiest fix to all this is just to add another entry to the loclist, so it's actually possible to navigate forwards in the locklist.

Reply:
Resolving. [We decided not to change the default behavior](https://github.com/vim/vim/pull/15841#pullrequestreview-2361372539), which means this test does not need to change.